### DISCUSSION TITLE
calculate year based on a date

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17550] [How to]
Hello,

In a date field, I need to obtain the year in another field, for example a text field.
Example: An account that was created on 12/05/2000, what would be the expression or way to obtain the years that that account has been registered in the system?
Based on the example in date format dd/mm/yyyy that account would be 24 years old.

**Comment by mariuszkrzaczkowski** [comment]
you need to create your own function in textparser
